src/concepts/proxy/mqtt/mqtt.md

```diff
@@ -0,0 +1,41 @@
+---
+shortTitle: Proxy
+---
+
+# MQTT Proxy
```

Comment:
we can add this example : https://github.com/aklivity/zilla-examples/tree/main/mqtt.proxy.jwt
similar to other proxies